test/test_ci_containment.rb

```diff
@@ -64,14 +39,16 @@ def setup
         }
         config.filter_sensitive_data('<%= Dir.pwd %>', :erb_pwd) { Dir.pwd }
       end
-      # Chdir to root, as Containment will set the working dir to PWD and this
-      # is slightly unwanted for tmpdir tests.
       Dir.chdir('/')
 
       @job_name = 'vivid_unstable_test'
       @image = PangeaImage.new('ubuntu', 'vivid')
 
-      VCR.turned_off { cleanup_container }
+      VCR.turned_off do
+        cleanup_container
+        image = Docker::Image.create(fromImage: 'ubuntu:vivid')
```

Comment:
Image creation and tagging should not be done when in playback mode. (see vcr_it in container upgrade test or smth)